backend/src/main/java/ai/verta/modeldb/experimentRun/FutureExperimentRunDAO.java

```diff
@@ -770,7 +770,13 @@ public FutureExperimentRunDAO(
         InternalFuture.supplyAsync(
             () -> {
               final var localQueryContext = new QueryFilterContext();
-              localQueryContext.getConditions().add("deleted = :deleted");
+              localQueryContext.getConditions().add("experiment_run.deleted = :deleted");
+              localQueryContext.getBinds().add(q -> q.bind("deleted", false));
+
+              localQueryContext.getConditions().add("p.deleted = :deleted");
+              localQueryContext.getBinds().add(q -> q.bind("deleted", false));
+
+              localQueryContext.getConditions().add("e.deleted = :deleted");
               localQueryContext.getBinds().add(q -> q.bind("deleted", false));
```

Comment:
You cannot bind to the same `deleted` name.

Reply:
Yahh, we can bind because it will replace `deleted` value. but good point i will take it a single entry. updating PR

Reply:
Fixed